Ccq/Console/UnitTwo.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace RLTSquare\Ccq\Console;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Psr\Log\LoggerInterface;
+use RLTSquare\Ccq\Api\Data\QueueDataInterface;
+use Magento\Framework\MessageQueue\PublisherInterface;
+
+class UnitTwo extends Command
+{
+    /**
+     *  number1
+     */
+    const FirstNumber = 'var1';
+    /**
+     * number2
+     */
+    const SecondNumber = 'var2';
+    /**
+     * @var PublisherInterface
+     */
+    protected $publisher;
+    /**
+     * @var QueueDataInterface
+     */
+    protected $queueData;
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
+
+    /**
+     * @param LoggerInterface $logger
+     * @param QueueDataInterface $queueData
+     * @param PublisherInterface $publisher
+     * @param string|null $name
+     */
+    public function __construct(
+
+        LoggerInterface    $logger,
+        QueueDataInterface $queueData,
+        PublisherInterface $publisher,
+        string             $name = null
+    )
+    {
+        $this->publisher = $publisher;
+        $this->queueData = $queueData;
+        $this->logger = $logger;
+        parent::__construct($name);
+
+    }
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('RLTSquare:sum:number');
+        $this->setDescription('My awesome Console command');
+
+        // Next line will add a new required parameter to our script
+        $this->addArgument(
+            self::FirstNumber,
+            null,
+            InputArgument::IS_ARRAY,
```

Comment:
Overlooked logic - The Symfony\Component\Console\Input\InputOption would be more suitable for task - see https://symfony.com/doc/current/console/input.html#using-command-arguments